src/server_manager/web_app/app.ts

```diff
@@ -342,7 +332,8 @@ export class App {
         messageKey = 'error-servers-removed';
         placeholder = 'serverNames';
       }
-      this.appRoot.showError(this.appRoot.localize(messageKey, placeholder, unsyncedServerNames));
+      this.notificationManager.showErrorRaw(
```

Comment:
Do you need showErrorRaw?
You could make the showError signature match the localize signature and take parameters.